Bottom: a2f94eb757d599b73edb2d5a9dc72456816a75fb
Top:    8ca4d63afa834bf4c2a839b0d440442364c2e710
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-07 00:06:08 +0100

TEMP TEST


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index ea77412..878cbf3 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -280,6 +280,7 @@ have_crosshair_attachments (void)
 static gboolean
 ghid_idle_cb (gpointer data)
 {
+  return;
   if (Settings.Mode == NO_MODE)
     SetMode (ARROW_MODE);
   ghid_mode_cursor (Settings.Mode);
